test/format/config_def.c

```diff
@@ -129,6 +129,13 @@ CONFIG configuration_list[] = {
   {"disk.mmap_all", "configure mmap operations (read and write)",
     C_BOOL, 5, 0, 0, V_GLOBAL_DISK_MMAP_ALL},
 
+/* Test format can only handle 32 tables so we use a maximum value of 32 here. */
+  {"file_manager.close_handle_minimum", "number of handles open before the file manager will look for handles to close",
+    0x0, 0, 32, 32, V_GLOBAL_FILE_MANAGER_CLOSE_HANDLE_MINIMUM},
```

Comment:
Why the default value is also 32? Should we use a smaller value for default?

Reply:
Hi Chenhao, thanks for picking this up, I have changed the maxrand value to 20 in this commit: f21c1943375f172583d6d324b3feab3e3a39ad06

Reply:
For default lets do 8!